dev/linters/yamllint.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+# shellcheck disable=SC2046
+# We want word splitting with find.
+bazel run //dev/linters:yamllint -- \
+      -d "{extends: relaxed, rules: {line-length: {max: 120}}}" \
+      --strict $(find . -not -path "./deploy/helm/kubecf/*" -name '*.yaml' -or -name '*.yml' )
```

Comment:
[nit] Missing type file only.
[Super nit] Extra space on the closing parenthesis.

Reply:
I think we still want to lint `values.yaml`.

Reply:
I'm wondering if it would worth running the `yamllint` on the rendered templates. It would require inverting the build step with the linting step on CI. Something like:

```
bazel run //dev/linters:yamllint -- ... <(helm template --name kubecf ...)
```

Ping @mook-as.

Reply:
Running linters on generated output can often be a bad idea (because we would often get generated output that would be different than how a human would write things from scratch, e.g. with very long lines).

Reply:
In this case, we should aim to generate human-like manifests. A linter on the generated output would help us with achieving that. It's a different case than running a linter on, let's say, gRPC generated code, although I'd argue that it may be subject to some pre-compile checks. The reason why I believe we should aim to generate human-like manifests is that it makes inspection easy for everyone from devs to final customers.

Reply:
Good catch, I have added `deploy/helm/kubecf/values.*`
We could always lint before and after build.